lib/src/signed_video_h26x_sign.c

```diff
@@ -574,6 +574,8 @@ signed_video_add_nalu_part_for_signing_with_timestamp(signed_video_t *self,
         // adding it to the SEI NALU. Now it is time to start a new GOP, that is, hash and add this
         // first NALU of the GOP.
         SV_THROW(hash_and_add(self, &nalu));
+        // Chek if Nalu has been used for linking before.
```

Comment:
Comment is fishy. Do you really check if linking has been done before? Aren't you actually updating the linked hash?